test/libsolidity/StandardCompiler.cpp

```diff
@@ -2232,6 +2232,31 @@ BOOST_AUTO_TEST_CASE(ethdebug_ethdebug_output)
 	}
 }
 
+BOOST_AUTO_TEST_CASE(ethdebug_output_instructions_smoketest)
```

Comment:
this could be parametrized over `bytecode` and `deployedBytecode` like

```suggestion
BOOST_DATA_TEST_CASE(ethdebug_output_instructions_smoketest, boost::unit_test::data::make(std::array{"deployedBytecode", "bytecode"}), sample)
```